pkg/subscriptions/subscriptions.go

```diff
@@ -155,6 +155,27 @@ func getMountsMap(path string) (string, string, error) { //nolint
 	return "", "", fmt.Errorf("unable to get host and container dir from path: %s", path)
 }
 
+// Return true iff the system is in FIPS mode as determined by reading
+// /proc/sys/crypto/fips_enabled and disableFips is not true.
+func shouldAddFIPSMounts(disableFips bool) bool {
+	if disableFips {
+		return false
+	}
```

Comment:
This makes no sense logically to me, why pass an argument just to exit without doing anything there.
IT seems like the caller should just perform this check, i.e. `if disableFips && !shouldAddFIPSMounts()`

Reply:
Fixed.